config/phpstan.neon

```diff
@@ -7,3 +7,6 @@ parameters:
 		- '#Plugin definitions cannot be altered.#'
 		- '#Missing cache backend declaration for performance.#'
 		- '#Plugin manager has cache backend specified but does not declare cache tags.#'
+	cognitive_complexity:
+		class: 50
+		function: 16
```

Comment:
I'm just reading a site audit made by Karhu and they refer a few times to too high cognitive complexity when it's over 15, so this is quite close to that. But they recommend to keep it below 10, which could be too low and cause too many errors.

Reply:
I think it was @mitrpaka who initially chose this?
I see in the plugin page they have example of 8.

So yeah, thing to discuss.

Reply:
I agree!